pynicotine/shares.py

```diff
@@ -262,6 +262,7 @@ def __init__(self, queue, share_groups, share_db_paths, init=False, rescan=True,
         self.processed_share_names = set()
         self.processed_share_paths = set()
         self.current_file_index = 0
+        self.current_folder_count = 0
```

Comment:
Local scope only inside `scan_shared_folder()` like `folder_paths`

Reply:
That will reset the count when we scan the next top-level folder.